src/modules/unit/components/UnitFilters.js

```diff
@@ -54,32 +64,33 @@ export class UnitFiltersComponent extends React.Component {
   }
 
   onMenuSelect = (key: string, value: string): void => {
-    this.setState({expandedFilter: null});
+    this.setState({ expandedFilter: null });
     this.props.updateFilter(key, value);
   }
 
   toggleExpandedFilter = (filter) => {
     const isFilterActive = filterEquals(filter, this.state.expandedFilter);
-    this.setState({expandedFilter: isFilterActive ? null : filter});
+    this.setState({ expandedFilter: isFilterActive ? null : filter });
   }
 
   render() {
-    const {filters, t} = this.props;
-    const {expandedFilter} = this.state;
+    const { filters, t } = this.props;
+    const { expandedFilter } = this.state;
 
-    const FilterOptions = ({filter}: {filter: UnitFilterProps}) => (
+    const FilterOptions = ({ filter }: {filter: UnitFilterProps}) => (
```

Comment:
I see that this was not changed now, but there is really no reason to create this component on every render, this should be separated to its own component outside render. Maybe add a fixme comment?

Reply:
7cb0c683a1d57371556a95f40250c99c43cd7f7c